README.md

```diff
@@ -15,6 +15,14 @@ Embedding applications via iframe also means that code from separate teams can b
 **JavaScript**
 
 ```js
+/*
+ * We require a few polyfills in order to support IE11.  These
+ * will be needed to be loaded by both the host and the client.
+ */
+import "@babel/polyfill";
+import "custom-event-polyfill/polyfill.js";
+import "url-polyfill";
```

Comment:
Thanks for documenting these!

Reply:
Should we add anything about `browserslist` and `preset-env` or maybe just a link on how to use the babel polyfills?

Reply:
I want to be careful about overspecifing too much.  Since the host app probably has lots of concerns.  Heck even babel polyfill itself probably not required (We have a different one in PureConnect for example).

But I am up for adding anything we feel may be helpful